release-notes/3.0/preview/api-diff/preview5/3.0-preview5_System.Runtime.Intrinsics.X86.md

```diff
@@ -0,0 +1,293 @@
+# System.Runtime.Intrinsics.X86
+
+``` diff
+ namespace System.Runtime.Intrinsics.X86 {
+     public abstract class Avx2 : Avx {
++        public unsafe static Vector256<short> ConvertToVector256Int16(byte* address);
++        public static Vector256<short> ConvertToVector256Int16(Vector128<byte> value);
++        public unsafe static Vector256<short> ConvertToVector256Int16(sbyte* address);
++        public unsafe static Vector256<int> ConvertToVector256Int32(byte* address);
++        public unsafe static Vector256<int> ConvertToVector256Int32(short* address);
++        public static Vector256<int> ConvertToVector256Int32(Vector128<byte> value);
++        public static Vector256<int> ConvertToVector256Int32(Vector128<ushort> value);
++        public unsafe static Vector256<int> ConvertToVector256Int32(sbyte* address);
++        public unsafe static Vector256<int> ConvertToVector256Int32(ushort* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(byte* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(short* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(int* address);
++        public static Vector256<long> ConvertToVector256Int64(Vector128<byte> value);
++        public static Vector256<long> ConvertToVector256Int64(Vector128<ushort> value);
++        public static Vector256<long> ConvertToVector256Int64(Vector128<uint> value);
++        public unsafe static Vector256<long> ConvertToVector256Int64(sbyte* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(ushort* address);
++        public unsafe static Vector256<long> ConvertToVector256Int64(uint* address);
+-        public static Vector256<ushort> ConvertToVector256UInt16(Vector128<byte> value);
```

Comment:
For reference, these were replaced with overloads that return signed `T`. This makes them match the native intrinsics and the Sse41 methods.